packages/logs/src/boot/logsPublicApi.spec.ts

```diff
@@ -227,6 +234,14 @@ describe('logs entry', () => {
       })
     })
 
+    it('logs a message with "logger" origin', () => {
+      updateExperimentalFeatures(['forward-logs'])
+      LOGS.logger.log('message')
+
+      expect(getLoggedMessage(0).message.origin).toEqual(ErrorSource.LOGGER)
+      resetExperimentalFeatures()
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏you should reset global state in a `afterEach` callback because if you don't and the test fails, the state won't be reset.

Reply:
Sure